src/aequitas/plotting.py

```diff
@@ -0,0 +1,1064 @@
+
+        label_position_values = list(group_table[group_metric].values)
+
+        lighter_coppers = self.__truncate_colormap('copper_r', min_value=0,
+                                                   max_value=0.65)
+
+        norm = colors.Normalize(vmin=group_table['group_size'].min(),
+                                vmax=group_table['group_size'].max())
+        mapping = cm.ScalarMappable(norm=norm, cmap=lighter_coppers)
+
+        # Lock absolute value metric plot x-axis to (0, 1)
+        ax_lim = 1
+        ax.set_xlim(0, ax_lim)
+
+        for attribute_name in attribute_names:
+
+            attribute_data = group_table.loc[
+                (group_table['attribute_name'] == attribute_name)]
+
+            values = attribute_data[group_metric].values
+            grp_sizes = attribute_data['group_size'].values
+
+            attribute_indices = np.arange(next_bar_height,
+                                          next_bar_height + attribute_data.shape[0],
+                                          step=height_of_bar)
+            attribute_tick_location = float((min(attribute_indices) + max(attribute_indices) + height_of_bar)) / 2
+
+            h_attribute = ax.barh(attribute_indices,
+                                  width=values,
+                                  # label=list(attribute_data['attribute_value'].values),
+                                  align='edge', edgecolor='grey')
+
+            label_colors = []
+            min_brightness = 0.55
+
+            for (i, bar), g_size in zip(enumerate(h_attribute), grp_sizes):
+                my_col = mapping.to_rgba(g_size)
+                bar.set_color(my_col)
+                label_colors.append(self.__brightness_threshold(
+                    my_col[:3], min_brightness, light_color=(1, 1, 1, 1)))
+
+            if label_dict:
+                labels = [label if label not in label_dict.keys() else
+                          label_dict[label] for label in
+                          attribute_data['attribute_value'].values]
+            else:
+                labels = attribute_data['attribute_value'].values
+
+            for y, label, value, text_color, g_size in zip(attribute_indices, labels,
+                                                   values, label_colors,
+                                                   grp_sizes):
+                next_position = label_position_values.pop(0)
+                group_label = f"{label} ({g_size:,})"
+
+                if ax_lim < 3:
+                    CHAR_PLACEHOLDER = 0.03
+                else:
+                    CHAR_PLACEHOLDER = 0.25
+
+                label_length = len(label) * CHAR_PLACEHOLDER
+                max_val_length = 7 * CHAR_PLACEHOLDER
+                indent_length = ax_lim * 0.025
+
+                # bar long enough for label, enough space after bar for value
+                if ((indent_length + label_length) < (next_position - indent_length)) & (
```

Comment:
to be sure ... do you _really_ mean to do the `&` -- bitwise and, as opposed to the boolean and?